src/modules/drawer.js

```diff
@@ -387,6 +405,11 @@ Drawer.propTypes = {
     className: PropTypes.string,
     dimmer: PropTypes.bool,
     isOpen: PropTypes.bool.isRequired,
+    isSticky: PropTypes.bool,
+    maxHeight: PropTypes.oneOfType([
+        PropTypes.number,
+        PropTypes.string,
+    ]),
```

Comment:
Let's just go with a number instead of both for `maxHeight`.

Reply:
Fixed